Changes

```diff
@@ -9,6 +9,10 @@ Working version
   can be used as a placeholder for a polymorphic function.
   (Stephen Dolan)
 
+- GPR#1247: M.(::) construction for expression and pattern
+  (and fix printing of (::) in toplevel)
```

Comment:
Changes specialist trick of the day: cutting lines early can avoid sentences starting with a parenthesis, to avoid confusion with the credit line. `and pattern (plus fixing of (::) in toplevel)`.